### ISSUE TITLE
Upgraded to latest pod, now delegates not firing

### ISSUE THREAD

**Issue by tspangler** [marcuswestin/WebViewJavascriptBridge#130]
The title says it all. I just upgraded to the latest pod from an ancient version, and now my UIWebView delegate methods aren't firing.

In my header file:
```objective-c
@interface MyViewController : UIViewController <UITextFieldDelegate,RestConnectionDelegate,UIWebViewDelegate> 

@property WebViewJavascriptBridge* bridge;
```

In my implementation file:
```objective-c
    NSLog(@"Setting up JavaScript bridge...");
    [WebViewJavascriptBridge enableLogging];
    
    _bridge = [WebViewJavascriptBridge bridgeForWebView:_register_web_view webViewDelegate:self handler:^(id data, WVJBResponseCallback responseCallback) {
        // yadda yadda
    }];
    
    NSLog(@"webView delegate is %@", _register_web_view.delegate.description);
    [_register_web_view loadRequest:requestObj];
```
That `NSLog` on the second-to-last line shows that the delegate is set properly:

`webView delegate is <WebViewJavascriptBridge: 0x17409e7d0>`

I have implemented all of UIWebView's delegate methods and included `NSLog` statements in each to confirm that the methods are not being fired.

In short, I've covered all my bases and I don't know what's going on. Any suggestions would be appreciated.

**Comment by tspangler** [comment]
Closing. Changing _bridge to self.bridge in the assignment fixed it.
